lib/transform.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
```

Comment:
this is required for the bindings module to work with browserify. Should go into a module and use the AST if we decide we want this approach.